packages/mui-material/src/ListItemText/ListItemText.test.js

```diff
@@ -155,6 +155,27 @@ describe('<ListItemText />', () => {
     expect(texts[1]).have.text('This is the secondary text');
   });
 
+  it('should use variant if provided', () => {
+    const { container } = render(
+      <ListItemText
+        primary="This is the primary text"
+        primaryTypographyProps={{ variant: 'h3' }}
+        secondary="This is the secondary text"
+        secondaryTypographyProps={{ variant: 'h4' }}
+      />,
+    );
+    expect(container.firstChild.firstChild).to.have.tagName('h3');
+    expect(container.firstChild.lastChild).to.have.tagName('h4');
```

Comment:
Can you use `getByText` query selector here and in below test? It's easier to use this for non-interactive elements (div, span etc) instead of accessing nodes. Also improves readability.
```suggestion
    expect(getByText('This is the primary text')).to.have.tagName('h3');
    expect(getByText('This is the secondary text')).to.have.tagName('h4');
```

Reply:
Good advice.